### PR TITLE
fix: Deprecate redundant Card component tokens and classes

### PR DESCRIPTION
**Pull request by magnuh**
- Use semantic classes (`s-bg`) directly instead of component token classes (`i-bg-$token`) where possible
- Deprecate redundant component tokens and classes

### CODE REVIEW

**Review by felicia-haggqvist — APPROVED** [review]

File: component-classes/index.js [hunk 1]
```diff
@@ -107,15 +107,15 @@ export const steps = {
...
-  cardShadow: 'rounded-8 i-shadow-$shadow-card hover:i-shadow-$shadow-card-hover hover:i-bg-$color-card-background-hover tap-highlight-transparent',
+  cardShadow: 'rounded-8 shadow-s hover:shadow-m hover:s-bg-subtle-hover tap-highlight-transparent',
   cardFlat: 'border-2 rounded-4',
-  cardFlatUnselected: 'i-bg-$color-card-flat-background i-border-$color-card-flat-border hover:i-bg-$color-card-flat-background-hover hover:i-border-$color-card-flat-border-hover active:i-bg-$color-card-flat-background-active active:i-border-$color-card-flat-border-active',
-  cardFlatSelected: 'i-border-$color-card-flat-border-selected i-bg-$color-card-flat-background-selected hover:i-bg-$color-card-flat-background-selected-hover hover:i-border-$color-card-flat-border-selected-hover active:i-border-$color-card-flat-border-active active:i-bg-$color-card-flat-background-active',
+  cardFlatUnselected: 's-bg s-border hover:s-bg-subtle-hover hover:s-border-hover active:s-bg-active active:s-border-active',
+  cardFlatSelected: 's-border-selected s-bg-selected hover:s-bg-selected-hover hover:s-border-selected-hover active:s-border-selected-active active:s-bg-selected-active',
```
Comment (BalbinaK) [inline component-classes/index.js:113]:
```suggestion
  cardFlatSelected: 's-border-selected s-bg-selected hover:s-bg-selected-hover hover:s-border-selected-hover active:s-border-active active:s-bg-active',
```

File: component-classes/index.js [hunk 2]
```diff
@@ -107,15 +107,15 @@ export const steps = {
...
   cardUnselected: 's-bg',
-  cardSelected: 'i-border-$color-card-border-selected i-bg-$color-card-background-selected hover:i-border-$color-card-border-selected-hover hover:i-bg-$color-card-background-selected-hover active:i-border-$color-card-border-selected-active',
-  cardOutline: 'active:i-border-$color-card-flat-border absolute rounded-8 inset-0 transition-all border-2',
-  cardOutlineUnselected: 'i-border-$color-card-border',
-  cardOutlineSelected: 'i-border-$color-card-border-selected hover:i-border-$color-card-border-selected-hover',
+  cardSelected: 's-border-selected s-bg-selected hover:s-border-selected-hover hover:s-bg-selected-hover active:s-border-selected-active',
```
Comment (BalbinaK) [inline component-classes/index.js:115]:
```suggestion
  cardSelected: 's-border-selected s-bg-selected hover:s-border-selected-hover hover:s-bg-selected-hover active:s-border-selected',
```

There's no such semantic token as border selected active, and the [Taxonomy sheet](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=1588196525&range=178:178) seems to set `color-card-border-selected` to `color-border-selected`

File: component-classes/index.js (hunk 1 above)
Comment (magnuh) [reply component-classes/index.js:113]: These should really be `selected-active`... so I'm looking in to adding those tokens. 😅

Comment (magnuh) [reply component-classes/index.js:113]: https://github.com/warp-ds/css/pull/145

File: component-classes/index.js (hunk 2 above)
Comment (magnuh) [reply component-classes/index.js:115]: https://github.com/warp-ds/css/pull/145

**Pull request merged**